### ISSUE TITLE
Gapsize not working

### ISSUE THREAD

**Issue by Heli515** [highcharts/highcharts-angular#78]
Hi,

I've hoping to break the line chart when there're bug gaps. However, 'gapsize' doesn't seem to be working. Here's my code:

```javascript
  charts = [{
    hcOptions: {
     title: { text: this.chartName },
     reflow: false,
     legend: {
       enabled: true,
       layout: 'vertical',
       align: 'left',
       verticalAlign: 'top',
       x:0,
       y:50,
       maxHeight: 150,
       floating: true,
       backgroundColor: '#FFFFFF',
       borderWidth: 1
     },
     credits: {
      enabled: false
    },
    plotOptions: {
      series: {
        gapUnit: `relative`,
        gapSize:1
      }
    },
    xAxis: {
      ordinal: false,
      title:  {
        enabled:true,
        text: this.highchartX
      }
    },
    yAxis:{
      title:  {
        enabled:true,
        text: this.highchartY
      }
    },
    scrollbar: {
      enabled: true,
      barBackgroundColor: '#0088e5',
      barBorderRadius: 7,
      barBorderWidth: 0,
      buttonBackgroundColor: 'gray',
      buttonBorderWidth: 0,
      buttonBorderRadius: 7,
      trackBackgroundColor: 'none',
      trackBorderWidth: 1,
      trackBorderRadius: 8,
      trackBorderColor: '#CCC'
    },
    connectNulls: false,
    boost: {
      allowForce: true,
      seriesThreshold:1
    },
    rangeSelector : {
      buttons: [
        {
          type: 'minute',
          count: 1,
          text: '1mi',
      },{
          type: 'hour',
          count: 1,
          text: '1hr',
      }, {
          type: 'day',
          count: 1,
          text: '1dy'
      }, {
          type: 'month',
          count: 1,
          text: '1mo'
      }, {
          type: 'year',
          count: 1,
          text: '1yr'
      }, {
          type: 'all',
          text: 'All'
      }],
      inputEnabled: false, // it supports only days
      selected : 4 // all
      },
    boostThreshold: 1,
    series: []
   },
    hcCallback: (chart: Highcharts.Chart) => { this.localChart = chart}
  }];
```

**Comment by KacperMadej** [comment]
Hi @Heli515 

For technical support please contact [Highcharts technical support](https://www.highcharts.com/support).

Are you sure this is a problem with the wrapper and not an general Highcharts issue? If you could create a demo in Highcharts showing the case as working and a demo in Angular showing the problem in the wrapper then it would clearer where the issue might be.

Also, hard to test the problem without data set, so I have used made up data that should show gaps.
Demo working fine: https://codesandbox.io/s/5x34364yqp

**Comment by Heli515** [comment]
> Hi @Heli515
> 
> For technical support please contact [Highcharts technical support](https://www.highcharts.com/support).
> 
> Are you sure this is a problem with the wrapper and not an general Highcharts issue? If you could create a demo in Highcharts showing the case as working and a demo in Angular showing the problem in the wrapper then it would clearer where the issue might be.
> 
> Also, hard to test the problem without data set, so I have used made up data that should show gaps.
> Demo working fine: https://codesandbox.io/s/5x34364yqp

Hi @KacperMadej ,

Thank you for you reply! After playing with it, I realized 'highchart-boost' package was what's causing the conflict. (https://www.npmjs.com/package/highcharts-boost) Chart is working fine now after deleting this package.

Thanks!
